server.js

```diff
@@ -44,8 +45,14 @@ app.post("/callback", async (req, res) => {
   }
 });
 
+app.get("/api/fromNumbers", async (req, res) => {
+  res.send(APPLICATION_FROM_NUMBERS);
+});
+
 app.get("/api/threads", async (req, res) => {
-  res.send(await db.getAllThreads());
+  let threads = await db.getAllThreads();
+  console.log(`threadss: ${JSON.stringify(threads)}`);
```

Comment:
💄  I think there's an extra `s` here. 

For the line above it can be useful to use `const` instead of `let` if you know the variable you're defining isn't going to change so that it isn't accidentally mutated later. It's not that important in this case but can be useful in big blocks of code.